File: app/app/src/main/java/com/spartano/tiendamovil/model/PerfilDataResponse.java
package com.spartano.tiendamovil.model;

public class PerfilDataResponse {
    Usuario usuario;
    int valoracion;
    int cantidadReseñas;
    int cantidadVentas;
    String reputacion;

    public String getReputacion() {
        return reputacion;
    }

    public void setReputacion(String reputacion) {
        this.reputacion = reputacion;
    }

    public Usuario getUsuario() {
        return usuario;
    }

    public void setUsuario(Usuario usuario) {
        this.usuario = usuario;
    }

    public int getValoracion() {
        return valoracion;
    }

    public void setValoracion(int valoracion) {
        this.valoracion = valoracion;
    }

    public int getCantidadReseñas() {
        return cantidadReseñas;
    }

    public void setCantidadReseñas(int cantidadReseñas) {
        this.cantidadReseñas = cantidadReseñas;
    }

    public int getCantidadVentas() {
        return cantidadVentas;
    }

    public void setCantidadVentas(int cantidadVentas) {
        this.cantidadVentas = cantidadVentas;
    }
}

File: api/TiendaMovil/Models/PerfilDataResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TiendaMovil.Models
{
    public class PerfilDataResponse
    {
        public Usuario Usuario { get; set; }
        public int Valoracion { get; set; }
        public int CantidadReseñas { get; set; }
        public int CantidadVentas { get; set; }
        public string Reputacion { get; set; }
    }
}
